draft-birkholz-scitt-architecture.md

```diff
@@ -483,10 +483,10 @@ TS implementations MUST indicate their support for registration policies and MUS
 
 Policy Name | Required `reg_info` attributes | Implementation
 ---|---|---
-TimeLimited | `register_by: uint` | Returns true if now () < register_by at registration time. The ledger MUST store the ledger time at registration along with the claim, and SHOULD indicate it in receipts
+TimeLimited | `register_by: uint` | Returns true if now () < `register_by` at registration time. The ledger MUST store the ledger time at registration along with the claim, and SHOULD indicate it in receipts. The value provided for `register_by` MUST be a Unix timestamp, i.e., a `uint` representing the number of seconds since 00:00:00 UTC, 1 January 1970, without leap seconds.
```

Comment:
"without leap seconds" is ambiguous.
Please refer to Section 3.4.2 of RFC 8949 for a precise definition; of course, you would limit this to unsigned integers (which is analog to the last sentence of the penultimate paragraph of that Section).
(See [my other comment](https://github.com/ietf-scitt/draft-birkholz-scitt-architecture/pull/41#issuecomment-1351160877) for how to make this evident in the CDDL too, based on Henk's comment.)

Reply:
The CDDL is now meaningful, reflecting your other comments. That makes this explanation redundant, however I'm inclined to keep a brief explanation to aid most readers, so I've updated the wording to reflect Section 3.4.2 of RFC 8949 as you suggest.